content/en/security/cloud_security_management/guide/public-accessibility-logic.md

```diff
@@ -205,6 +214,121 @@ An [SQS Queue][32] (`aws_sqs_queue`) is considered publicly accessible if:
+|The rules which match the above properties combine with any other `Deny` rules of higher priority to open at least one port to the internet | See [here][39] for how Azure combines security group rules to calculate access |
+
+For more details on how Azure NSGs allow and deny internet access for a resource, see [here][40].
+
+### Azure Virtual Machine Instance
+
+A Virtual Machine Instance (`azure_virtual_machine_instance`) is considered publicly accessible if:
+
+* _Network Security Group allowing public access:_
+
+| Criteria | Explanation |
+|----------|-------------|
+|The virtual machine instance has a public IP address attached to its network interface | A public IP is required for internet access to a virtual machine instance |
+|The virtual machine instance has a network security group granting public access attached to its network interface | Refer to the Azure Network Security Group (NSG) section for how a network can grant public access |
+
+***OR***
+
+* _Public IP with SKU "Basic":_
+
+| Criteria | Explanation |
+|----------|-------------|
+|The virtual machine instance has a public IP address with SKU Basic attached to its network interface | A public IP address with SKU basic is open by default (see [here][41]) |
+|The virtual machine instance has no attached network security groups | If no network security groups are attached, then there are no rules blocking access through the open public IP address |
+
+Learn more about Azure Virtual Machine Instances and public access [here][42].
+
+### Azure Managed Disk
+
+A Managed Disk (`azure_managed_disk`) is considered publicly accessible if:
+
+| Criteria | Explanation |
+|----------|-------------|
+|The managed disk’s `network_access_policy` is set to `AllowAll` | This setting means that “The disk can be exported or uploaded to from any network” (see [here][43]) |
+|The managed disk’s virtual machine instance is publicly accessible | Refer to the Azure Virtual Machine Instance section above for how a virtual machine is considered publicly accessible |
+
+For more information about restricting import/export access for Azure Managed Disks, refer [here][44].
+
+### Azure Storage Account
+
+A Storage Account (`azure_storage_account`) grants public access if:
+
+| Criteria | Explanation |
+|----------|-------------|
+|The storage account’s `allow_blob_public_access` attribute is set to `true`, or is nonexistent | This means that the account allows public internet access to Azure Blob Storage |
+
+See more about configuring anonymous read access with Azure Storage Accounts [here][45].
+
+### Azure Storage Blob Container
+
+A Storage Blob Container (`azure_storage_blob_container`) is considered publicly accessible if:
+
+| Criteria | Explanation |
+|----------|-------------|
+|The storage blob container’s `public_access` attribute is not set to `None` | This means that the account allows public internet access to Azure Blob Storage |
```

Comment:
double negation makes it hard to read, let's use one of the enum values from https://learn.microsoft.com/en-us/rest/api/storageservices/list-containers2?tabs=microsoft-entra-id#response-body which can be `blob` or `container`